components/date-picker/locale/ru_RU.ts

```diff
@@ -16,6 +16,21 @@ const locale: PickerLocale = {
     rangeYearPlaceholder: ['Начальный год', 'Год окончания'],
     rangeMonthPlaceholder: ['Начальный месяц', 'Конечный месяц'],
     rangeWeekPlaceholder: ['Начальная неделя', 'Конечная неделя'],
+    shortWeekDays: ['Вс', 'Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб'],
+    shortMonths: [
+      'Янв',
+      'Февр',
+      'Март',
+      'Апр',
+      'Май',
+      'Июнь',
+      'Июль',
+      'Авг',
+      'Сент',
+      'Окт',
+      'Нояб',
+      'Дек',
+    ],
```

Comment:
It is better to send PR to dayjs: https://github.com/iamkun/dayjs/blob/a9d7d0398d22ebd4bfc3812ca0134a97606d54d9/src/locale/ru.js#L51-L52

Reply:
ping

Reply:
Hi, I checked the ru locale in day js, and I saw that it has all of these words, but when im using the ru locale in antd config provider, I get next: 
<img width="294" alt="image" src="https://github.com/ant-design/ant-design/assets/48078456/91ae9098-1c0c-47a8-be83-d750a143db3f">

Reply:
Could you check https://github.com/ant-design/ant-design/issues/43997#issuecomment-1663552566 ?

Reply:
lol, i checked and as i understood i still need to import 'dayjs/locale/ru'. I didnt expect this, I thought using the locale from antd was enough.
taking this opportunity, I want to ask a stupid question - what is the difference between imports from 'antd/es/*' and 'antd/lib/*'

Reply:
`antd/es` use `import` and `export` as es module, `antd/lib` are commonjs module.